app/_src/deck/guides/defaults.md

```diff
@@ -508,6 +508,28 @@ _info:
 For all available properties, see the
 [Target object](/gateway/latest/admin-api/#target-object) documentation.
 
+{% endnavtab %}
+{% navtab Plugins %}
+
+Use the Kong Admin API `/schemas` endpoint to find default values:
+
+<!-- codeblock tabs -->
+{% navtabs codeblock %}
+{% navtab cURL %}
+```sh
+curl -i http://localhost:8001/schemas/plugins/<plugin-name>
+```
+{% endnavtab %}
+{% navtab HTTPie %}
```

Comment:
I thought we were removing HTTPie? But maybe I'm misremembering.

Reply:
nice catch. I'll remove it from here.